cmd/vic-machine/create/create.go

```diff
@@ -1097,7 +1097,12 @@ func (c *Create) generateCertificates(server bool, client bool) ([]byte, *certif
 
 	var certs []byte
 	// generate the certs and keys with names conforming the default the docker client expects
-	err := os.MkdirAll(c.certPath, 0700)
+	files, err := ioutil.ReadDir(c.certPath)
+	if len(files) > 0 {
+		return nil, nil, fmt.Errorf("Specified directory to store certificates, \"%s\", is not empty. Please specify a new path in which to store generated certificates using --cert-path or remove the contents of \"%s\" and run vic-machine again.", c.certPath, c.certPath)
```

Comment:
I think we can be a little more concise - if certPath is long this error message is going to become unwieldy. It's also imperative so drop the "Please".
`("\"%s\" is not empty. Remove the contents or specify a different path for generated certificates via -cert-path", c.certPath)`

I also agree with @emlin that if we're not overwriting certs then it would make sense to bail immediately after validation of existing certificates fails, with a similar, but specific error message:
`("cannot validate certificates in \"%s\" for use with this VCH. Specify a different path via -cert-path, or remove existing certificates to generate new ones", c.certPath)`

Agree completely that silently overwriting is an issue and that we should change that behaviour unless --force is specified.
Not sure whether `--force` should overwrite, or use unvalidated certs.... It's a very coarse mechanism.